src/ui/storage/commands/persistDeviceOptions.ts

```diff
@@ -22,6 +22,7 @@ const persistDeviceOptions = async (
           switch (userDefine.key) {
             case UserDefineKey.BINDING_PHRASE:
               await storage.setBindingPhrase(userDefine.value);
+              await storage.updateBindingPhraseHistory(userDefine.value);
```

Comment:
If the binding phrase history is updated here, it will store a new binding phrase on every key press, it would be better to save the binding phrase to history when the firmware is built.